implementations/rust/ockam/ockam_core/src/routing/address.rs

```diff
@@ -159,6 +159,13 @@ impl Address {
     pub fn random(tt: u8) -> Self {
         Self { tt, ..random() }
     }
+
+    /// Create a new address with a suffix
+    pub fn suffix<S: Into<String>>(&self, s: S) -> Self {
+        let mut inner = self.inner.clone();
+        inner.append(&mut s.into().as_bytes().to_vec());
+        Self { inner, tt: self.tt }
+    }
```

Comment:
This isn't used anymore but I don't know if I should remove it again. Is this conceivably in some other setting?

Reply:
I heard @antoinevg wanted suffixes/prefixes for addresses. I personally think addresses should be random where possible.

Reply:
I don't think that was addresses - it was a workaround to deal with the fact that we couldn't block on the return of stream names from elixir:

https://github.com/ockam-network/ockam/commit/931343473032a16a572ea35e0475af48ae175821#diff-408f7d99d304bd5499de15a5e8c41f54eb5cfa3e6b507a20b055764c44066f61R12-R14

The point is probably moot now anyway so don't hold onto that function for my sake!

Reply:
Alright, i'll remove it again then